src/parse.py

```diff
@@ -3,12 +3,27 @@
 lua = LuaRuntime(unpack_returned_tuples=True)
 
 def eval_string(string):
+  # change stuff like this:
+  # local unitName = Spring.I18N('units.names.armaca')
+  # description = Spring.I18N('units.heap', { name = unitName }),
+  # to this:
+  # local unitName = "units.names.armaca"
+  # description = "units.heap.armsomethingsomething",
+  unit_name = string.partition('\n')[0].replace("local unitName = ", "")
+  if "Spring.I18N('" in string:
+    unit_name = unit_name.replace("Spring.I18N('", "\"").replace("')", "\"", 1)
+    string = string \
+      .replace("Spring.I18N('", "\"") \
+      .replace("', { name = unitName }", "." + unit_name.split(".")[-1].replace("\"", "") + "'") \
+      .replace("')", "\"")
+
   try:
     data = lua.execute(string)
     return open_unit_table(data)
   except lupa._lupa.LuaError as e:
     if "attempt to index a nil value" in e.args[0]:
       return None
+    print(e)
```

Comment:
The `print(e)` shouldn't be needed as `raise` will raise the original error and that will contain the error message. If this isn't the case we should throw an exception with the error message as part of it instead of printing the value.

Reply:
Interestingly enough nothing reached console in this case. Dunno why.

Reply:
My bad, all is well, that specific error was not supposed to go anywhere.